metricbeat-tests/mysql_test.go

```diff
@@ -2,21 +2,33 @@ package main
 
 import (
 	"github.com/DATA-DOG/godog"
-	"github.com/elastic/metricbeat-tests-poc/cli/services"
+	log "github.com/sirupsen/logrus"
 )
 
-var mysqlService services.Service
+var mysqlTestSuite MetricbeatTestSuite
 
 func MySQLFeatureContext(s *godog.Suite) {
 	s.Step(`^MySQL "([^"]*)" is running for metricbeat "([^"]*)"$`, mySQLIsRunningForMetricbeat)
 	s.Step(`^metricbeat "([^"]*)" is installed and configured for MySQL module$`, metricbeatIsInstalledAndConfiguredForMySQLModule)
 	s.Step(`^there are no errors in the "([^"]*)" index$`, thereAreNoErrorsInTheIndex)
+
+	s.BeforeScenario(func(interface{}) {
+		log.Debug("Before scenario...")
+		mysqlTestSuite = MetricbeatTestSuite{}
+	})
+	s.AfterScenario(func(interface{}, error) {
+		log.Debug("After scenario...")
+		mysqlTestSuite.CleanUp()
+	})
```

Comment:
Same code is being added to all feature contexts, I think this should be part of some helper.